tests/integration/annotation_import/test_send_to_annotate_mea.py

```diff
@@ -13,15 +14,38 @@ def test_send_to_annotate_from_model(client, configured_project,
     [destination_project, _] = project_with_ontology
 
     queues = destination_project.task_queues()
-    initial_labeling_task = next(
-        q for q in queues if q.name == "Initial labeling task")
+    initial_review_task = next(
+        q for q in queues if q.name == "Initial review task")
+
+    # build an ontology mapping using the top level tools and classifications
+    source_ontology_builder = OntologyBuilder.from_project(configured_project)
+    feature_schema_ids = list(
+        tool.feature_schema_id for tool in source_ontology_builder.tools)
+    # create a dictionary of feature schema id to itself
+    ontology_mapping = dict(zip(feature_schema_ids, feature_schema_ids))
+
+    classification_feature_schema_ids = list(
+        classification.feature_schema_id
+        for classification in source_ontology_builder.classifications)
+    # create a dictionary of feature schema id to itself
+    classification_ontology_mapping = dict(
+        zip(classification_feature_schema_ids,
+            classification_feature_schema_ids))
+
+    # combine the two ontology mappings
+    ontology_mapping.update(classification_ontology_mapping)
 
     task = model_run.send_to_annotate_from_model(
         destination_project_id=destination_project.uid,
         batch_name="batch",
         data_rows=UniqueIds(data_row_ids),
-        task_queue_id=initial_labeling_task.uid,
-        params={})
+        task_queue_id=initial_review_task.uid,
+        params={
+            "predictions_ontology_mapping":
+                ontology_mapping,
```

Comment:
so this is basically {id1: id1, id2: id2... } right? Kind of weird mapping to itself. Should be test a case when we have like {id1: id2, id3: id4} ?

Reply:
This is one of the valid use-cases. I can add additional tests as a follow-up